build.gradle

```diff
@@ -53,18 +53,6 @@ targetCompatibility = '1.8'
 sourceCompatibility = '1.8'
 
 if (!project.hasProperty("android")) {
-    jar {
```

Comment:
We won't need this now we're using the `maven-publish` plugin (which explicitly sets versions).